codegen/src/main/twirl/templates/ScalaServer/Handler.scala.txt

```diff
@@ -7,10 +7,12 @@
 @akka.grpc.gen.Constants.DoNotEditComment
 package @service.packageName
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ ExecutionContext, Future }
```

Comment:
We avoided importing `Future` here and instead spelled it out below to make sure the generated code also works when there is an object called `Future` in the proto definition.

Reply:
My bad - got a bit carried away optimising imports, and should have left the patch minimal. Will revert those bits.

Reply:
Have reverted this change.